lib/reform/form/validation/unique_validator.rb

```diff
@@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
     model = form.model_for_property(attribute)
 
     # search for models with attribute equals to form field value
-    query = model.class.where(attribute => value)
+    if options[:case_sensitive] == false
+      downcased_value = value.nil? ? value : value.downcase
```

Comment:
Do you have any suggestions for handling unicode downcase/upcase in earlier versions of Ruby? Matz's suggestion seems to be to use the unicode gem: https://github.com/lang/unicode_utils

http://blade.nagaokaut.ac.jp/cgi-bin/scat.rb/ruby/ruby-core/26651
http://stackoverflow.com/questions/1910573/ruby-1-9-how-can-i-properly-upcase-downcase-multibyte-strings